system/Config/BaseService.php

```diff
@@ -11,6 +11,20 @@
 
 namespace CodeIgniter\Config;
 
+use Config\Cache;
+use Config\App;
+use Config\Encryption;
+use Config\Exceptions as ConfigExceptions;
+use Config\Filters as ConfigFilters;
+use Config\Format as ConfigFormat;
+use Config\Honeypot as ConfigHoneyPot;
+use Config\Images;
+use Config\Migrations;
+use Config\Pager as ConfigPager;
+use Config\View as ConfigView;
+use CodeIgniter\HTTP\Request;
+use Config\Toolbar as ConfigToolbar;
+use Config\Validation as ConfigValidation;
```

Comment:
sort alpha

Reply:
It seems already sortered:

```bash
➜  CodeIgniter4 git:(fix-rector-notice) ✗ php-cs-fixer fix system/Config/BaseConfig.php --rules=ordered_imports 
Loaded config default.
Using cache file ".php_cs.cache".
```

Reply:
Sort alphabetically. `Config\App` should come before `Config\Cache`. `CodeIgniter\HTTP\Request` is misplaced. `Config\View` is misplaced too.

Reply:
fix